.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,131 @@
+name: cloud-workflow
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+########## INSYNC-CHECKIN ##########
+
+  insync-checkin:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: |
+              echo "MAGMA_ROOT=$(pwd)" >> $GITHUB_ENV
+#      - build/determinator:
+#          <<: *all_gateways_build_verify
+    - name: Run apt-get update
+      run: sudo apt-get update
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: Run build.py
+      run: |
+          cd ${MAGMA_ROOT}/orc8r/cloud/docker
+          python3 build.py --generate
+          sudo chown -R $USER $MAGMA_ROOT/*
+          git add .
+          git status
+          git diff-index --quiet HEAD
+#    - magma_slack_notify
+
+########## CLOUD-LINT ##########
+
+  cloud-lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: |
+              echo "MAGMA_ROOT=$(pwd)" >> $GITHUB_ENV
+#      - build/determinator:
+#          <<: *all_gateways_build_verify
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: Lint cloud Go code
+      run: |
+          cd ${MAGMA_ROOT}/orc8r/cloud/docker
+          python3 build.py --lint
+    - name: Generate test coverage
+      run: |
+          cd ${MAGMA_ROOT}/orc8r/cloud/docker
+          python3 build.py --coverage
+#    - codecov/upload:
+#          file: /home/circleci/project/orc8r/cloud/coverage/all.gocov
```

Comment:
I'm not sure if this is needed? Does the codecov Github action currently work as a replacement for this job? Is this adding additional details not captured by the other action run? Or was this specifically needed in Circle-CI as there was no Codecov integration?

Reply:
You are right,,, seems like we already receive reports from Codecov via bot. 
Will remove these comments for now but will also double check with Tim